asserts/checks.py

```diff
@@ -33,17 +31,17 @@ def cpu_temp_check(data):
     )
 
 
-def batt_temp_check(data):
+def temp_check(data):
```

Comment:
for coherence purposes and avoid confusion of language, functions names should start with a verb
in this case I would name it VERB_SOMETHING, so check_temperature
also avoid trying to cut words like if its temperature dont cut it to temp as it will just be more confusion, temp could be temporary, temporal, tempucha..

Reply:
Good point, thanks! I'll fix it along with the other functions in that file.